### DISCUSSION TITLE
How can i logout from copilot extension?

### DISCUSSION THREAD

**Discussion by yukiyohure** [community/community#6823] [Copilot Conversations] [answered]
I have two Github account and I got an invitation from GitHub copilot for the accountA.
However, I was accidentally authorized with the accountB to use Github copilot which does not have an invitation.
How can I log out from the accountB and get authorized with the accountA?

Thank you.

**Comment by andrewlayman** [comment]
Can you please give us more details?  When you say "authorized with accountB," can you give us more details about the authorization?

**Comment by anwerskk** [comment]
I have the same problem. I was using my personal GitHub account when I authorized GitHub to access Visual Studio Code. But then I realized that the invite that I received to use copilot was in my official GitHub account. Now I don't have any idea how to unauthorize the old account and authorize the new account.

**Comment by securay** [comment, marked as answer]
In the account icon (bottom-left button in vscode) in the menu click on sign out button:
![image](https://user-images.githubusercontent.com/3249219/139896304-a376efc0-2157-4ea7-b74f-c74ca0368bb1.png)

**Comment by fs-srogers** [comment]
I don't see a way to sign back in with the authorized copilot account?

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
